src/components/Settings/Notifications/NotificationsPushover/index.tsx

```diff
@@ -41,13 +41,13 @@ const NotificationsPushover: React.FC = () => {
     accessToken: Yup.string()
       .required(intl.formatMessage(messages.validationAccessTokenRequired))
       .matches(
-        /^a[A-Za-z0-9]{29}$/,
+        /^[a-z\d]{30}$/i,
```

Comment:
There is no chance for capital letters?

Reply:
This does a case-insensitive match 😄

Reply:
Oh I didn't notice the flag. I saw you used \d but didn't see \w so assumed it was only lowercase.

Reply:
`\w` also includes underscores, which is why I didn't use it here. 😉